dashboard/app/models/course.rb

```diff
@@ -55,6 +56,14 @@ def localized_version_title
     I18n.t("data.course.name.#{name}.version_title", default: version_year)
   end
 
+  # Any course with a plc_course or no family_name is considered stable.
+  # All other courses must specify an is_stable boolean property.
+  def stable_or_default
```

Comment:
Looks like a comment of mine disappeared -- this name seems confusing because it is redundant, because "default" means "latest stable". Can we call it `stable?` instead?

Reply:
yeah, good call! i'll update the naming now